### PR TITLE
OSDOCS-4336: Content audit for Requesting CRI-O and Kubelet profiling data by using the Node Observability Operator

### PR DESCRIPTION
**Pull request by ahardin-rh**
Issue:
https://issues.redhat.com/browse/OSDOCS-4336

Version(s):
4.11+

Link to docs preview:
https://58208--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/node-observability-operator.html

### CODE REVIEW

File: modules/node-observability-install-cli.adoc [hunk 1]
```diff
@@ -49,7 +49,7 @@ metadata:
   namespace: node-observability-operator
 spec:
   targetNamespaces:
-  - node-observability-operator
+ []
```
Comment (alebedev87) [inline modules/node-observability-install-cli.adoc:52]: I believe the empty list should be in the same line as the key, so:
`targetNamespaces: []`

**Comment by ahardin-rh** [comment]
@sunilcio Can you please provide QE review? Thanks!

**Pull request merged**